atomic_dbus.py

```diff
@@ -374,6 +378,40 @@ def _ScanSetup(self, scan_targets, scanner, scan_type, rootfs, _all, images, con
     def Scan(self, scan_targets, scanner, scan_type, rootfs, _all, images, containers):
         return self._ScanSetup(scan_targets, scanner, scan_type, rootfs, _all, images, containers).scan()
 
+    @slip.dbus.polkit.require_auth("org.atomic.read")
+    @dbus.service.method("org.atomic", in_signature='ssss', out_signature= 's')
+    # sudo busctl --system call org.atomic /org/atomic/object org.atomic ActiveScans
+    # Only support scanning one image at a time via the async method
+    # Same with rootfs
+    def ScanAsync(self, scan_id, scanner=None, scan_type=None, rootfs=None):
```

Comment:
I don't think default arguments make sense on D-Bus. Also, D-Bus doesn't have the concept of `None`. Please handle passing empty strings instead of `None`, so that cockpit doesn't have to look up available scanners and scan_types (each of those have a default, right?)